src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -554,6 +554,12 @@ public class JavadocMethodCheck extends AbstractCheck {
      */
     public static final String MSG_DUPLICATE_TAG = "javadoc.duplicateTag";
 
+    /** The Generic start. */
+    private static final String GENERIC_START = "<";
```

Comment:
Not a generic.
It is html element syntax https://www.w3.org/TR/2012/WD-html-markup-20120329/syntax.html#doctype-syntax
Even them do not name it and keep it as `A ">" character`.
Name it element_start. In javadoc mention "html element start symbol"

Reply:
@Kevin222004 , this is easy fix, please do it sooner.

Reply:
please split PR into two commits, first commit, changing character to String.
second commit actual pitest fix.

Reply:
done

Please re-run failing ci